frontend/src/styles/colors.ts

```diff
@@ -5,11 +5,13 @@ const white = '#FFFFFF';
 const lightGray = '#D9D9D9';
 const gray = '#8A8A8A';
 const silver = '#BFBFBF';
+const oliveDrab = '#6CA12B';
 
 export const Colors = {
   primary: cornflowerBlue,
   onPrimary: white,
   divider: lightGray,
   header: gray,
   shadow: silver,
+  button: oliveDrab,
```

Comment:
IMO to jest zbędne. Główny przycisk posiada już odpowiednie stylowanie (sam je robiłeś :D). Wykorzystaj sobie propsa `htmlType="submit"`